hack/run-dockershim-critest.sh

```diff
@@ -39,11 +44,9 @@ sleep 10
 # Skip reopen container log test because docker doesn't support it.
 # Skip runtime should support execSync with timeout because docker doesn't
 # support it.
-if [ "$arch" == x86_64 ]; then
-	critest -ginkgo.skip="runtime should support reopening container log|runtime should support execSync with timeout" -parallel 8
-else
-	critest -ginkgo.skip="runtime should support reopening container log|runtime should support execSync with timeout|runtime should support SupplementalGroups" -parallel 8
```

Comment:
sure you can add this test back in when not x86_64?

Reply:
sure, done